packages/kit/src/runtime/app/forms.js

```diff
@@ -49,9 +50,10 @@ export function enhance(form, submit = () => {}) {
 
 		const action = new URL(
 			// We can't do submitter.formAction directly because that property is always set
+			// We do form.cloneNode() for avoid DOM clobbering - https://github.com/sveltejs/kit/issues/7593
 			event.submitter?.hasAttribute('formaction')
 				? /** @type {HTMLButtonElement | HTMLInputElement} */ (event.submitter).formAction
-				: form.action
+				: /** @type {HTMLFormElement} */ (form.cloneNode()).action
```

Comment:
Someone could conceivably have an `<input name="cloneNode">` which would still break this, no?

What are the performance implications of this clone? I'm hoping not huge, because it's not a deep clone.

Reply:
> Someone could conceivably have an <input name="cloneNode"> which would still break this, no?

That's a good question. We'll call it through the prototype as well.

> What are the performance implications of this clone? I'm hoping not huge, because it's not a deep clone.

Hopefully this will not affect performance too much, because as noted above it is not a deep cloning.

Reply:
Not remotely worried about the performance impact — if it was more than a fraction of a millisecond I'd be astonished (tried it just now with some `console.time(...)` calls and yep, it's basically free)